index.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use Twig\Loader\FilesystemLoader;
+use Twig\Environment;
+
+require __DIR__ . '/vendor/autoload.php';
+
+$loader = new FilesystemLoader("template");
+$view = new Environment($loader);
+
+$user = new UserClass\User("32", "Shnapdozer", "retel207@gmail.com", "sdgsgseg");
+$comment = new CommentClass\Comment($user, "Shnapdozer comments");
+
+$user1 = new UserClass\User("12", "Rewood", "retel207@gmail.com", "sdgsgseg");
+$comment1 = new CommentClass\Comment($user1, "Rewood comments");
+
+$comments = [ $comment, $comment1];
+
+if (!empty($_POST)) {
+    if (isset($_POST['id'])) {
```

Comment:
Можно короче
```suggestion
    $id = trim($_POST["id"] ?? null);
```
И лучше даже без trim, т.к. лишние пробелы должны проверяться валидацией
```suggestion
    $id = $_POST["id"] ?? null;
```